src/redux/store.js

```diff
@@ -0,0 +1,13 @@
+import {createStore, applyMiddleware, combineReducers} from "redux";
+import thunk from "redux-thunk";
+import modalReducer from "./modal/reducer";
+import movieReducer from "./movie/reducer";
+
+const rootReducer = combineReducers({
+    movieReducer,
+    modalReducer
```

Comment:
I think there is no much sense in adding `...Reducer` to the store names. seems that simple names like `movie` and `modal` will suffice.

Reply:
done